distributed/cli/tests/test_dask_remote.py

```diff
@@ -0,0 +1,7 @@
+from distributed.utils_test import popen
+
+
+def test_dask_remote():
+    with popen(['dask-remote', '--port', '9999']) as remote:
```

Comment:
@mrocklin any ideas on how this could be tested better? I was hoping their was a way to test if TCP communication is working.